### ISSUE TITLE
Need to wire up Email and Text fields of Thank you screen 

### ISSUE THREAD

**Issue by grundyoso** [flomio/bitflo#4]
Just like GoPaymentPlus with Parse table support.

**Comment by allynalford** [comment]
Will this be via the parse layer or native?

I'd imagine we'd use the layer.

On Tue, Jan 13, 2015 at 11:16 AM, Richard Grundy notifications@github.com
wrote:

> Just like GoPaymentPlus with Parse table support.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/flomio/bitflo/issues/4.

## 

Allyn Alford

**Comment by grundyoso** [comment]
it would be via the Parse layer. Boris did this on another app we built called GoPaymentsPlus using Parse integrated to their STMP service and Twilio, respectively.

**Comment by allynalford** [comment]
Yeah I've seen that one. That's why I was asking so I'd know if I'd need to
setup a method for it. I can just re-use that code.
